src/main/java/String06.java

```diff
@@ -0,0 +1,27 @@
+public class String06 {
+    public String shortcutString(String str) {
+        for (int i = 0; i < str.length() - 1; i++) {
```

Comment:
Hơi phức tạp ;)

Reply:
Thuật toán:
- nếu có 2 kí tự giống nhau đứng sát nhau count = 2, xóa 1 kí tự
- kiểm tra xem các kí tự sau đó có giống kí tự đó không nếu có thì xóa và  count ++;
- chèn count vào sau kí tự đó

Reply:
Done